### ISSUE TITLE
Question about the loss function

### ISSUE THREAD

**Issue by xiaojingli** [elanmart/psmm#1]
Hi Marcin, thank you for your code!
I'm confusing about the loss part. In the original paper the author stated the cross-entropy loss and the mixture model loss. 
<img width="494" alt="屏幕快照 2020-02-11 下午4 56 55" src="https://user-images.githubusercontent.com/11477264/74253612-980b6600-4cef-11ea-99b0-b4c45a102ebb.png">

But I noticed that in this implementation you only use `loss = (-log_probas).mean()` which is the negative log of the sum of pointer log-probablity and vocab-probability. 

So is `cross-entropy loss + mixture model loss`= `mixture model loss`? Could you explain it in more details?

Besides, the link to your blog seems not valid anymore...

Thanks for your help!
